src/server/services/wms/qgswmsgetstyles.cpp

```diff
@@ -50,14 +58,137 @@ namespace QgsWms
                                     QStringLiteral( "Layers is mandatory for GetStyles operation" ) );
     }
 
-    QStringList layersList = layersName.split( QStringLiteral( "," ), QString::SkipEmptyParts );
-    if ( layersList.size() < 1 )
+    QStringList layerList = layersName.split( QStringLiteral( "," ), QString::SkipEmptyParts );
```

Comment:
.split(',') is much faster - in general any qstring operation which can be replaced with a character based version will be much more performant